src/app/components/line-chart/line-chart.component.ts

```diff
@@ -0,0 +1,237 @@
+import { Component, OnInit } from '@angular/core';
+import { ObjectUnsubscribedError } from 'rxjs';
+
+
+
+@Component({
+  selector: 'app-line-chart',
+  templateUrl: './line-chart.component.html',
+  styleUrls: ['./line-chart.component.scss']
+})
+export class LineChartComponent implements OnInit {
+
+  multiOne;
+  routeNames: any[];
+  view: any[] = [700, 300];
+
+  // options
+  legend: boolean = true;
+  showLabels: boolean = true;
+  animations: boolean = true;
+  xAxis: boolean = true;
+  yAxis: boolean = true;
+  showYAxisLabel: boolean = true;
+  showXAxisLabel: boolean = true;
+  xAxisLabel: string = 'Request';
+  yAxisLabel: string = 'Time (Miliseconds)';
+  timeline: boolean = true;
+  selectedRoute;
+  colorScheme = {
```

Comment:
Can this be moved into CSS?

Reply:
This is the default and most common way to provide color configuration to the chart, from preliminary research it seems CSS is not possible.